applications/crossbar/priv/api/swagger.json

```diff
@@ -12702,37 +12702,9 @@
         "version": "2.0"
     },
     "paths": {
-        "/v2/about": {
-            "get": {}
-        },
```

Comment:
@jamesaimonetti yep, we know that.
how about using module attributes to mark these ? something like 
```
-api(root).
```
and then we could use Module:module_info(attributes) and look for `api` ?